FunctionCMakeDoxygenFilterCompile.cmake

```diff
@@ -1,11 +1,11 @@
 #!
 #! \brief Download and compile a CMake doxygen input filter
 #!
-#! \param OUT <out-file> (optional) Supply an absolute filename for
```

Comment:
What was the reason for removing this? The `<` and `>` characters should actually be escaped like `&lt;` and `&gt;`.